### ISSUE TITLE
[FR] Extendable exporter management

### ISSUE THREAD

**Issue by jiekun** [deepflowio/deepflow#4230]
### Search before asking

- [X] I had searched in the [issues](https://github.com/deepflowio/deepflow/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Currently we have otlp exporter and some other exporters in developing. However, we found some pain point contributing exporter to the community.

In order to provide an exporter that could be used by different people in different scenario, many params must be added to the configuration. For example:
  - when working with `log` exporter, which convert flow_log to plaintext log in one line, the log format is hard to control and match different requirement. It could be in json style, or seperated by `,`, or seperated by `|`. And log content / header field key can be different, like `ts=2023-09-13 09:50:00`, or `ts=2023-09-13 09:50:00.023`, or `timestamp=16323423432`, or `datetime=20230913095000`.

TLDR: You cannot provide a configuration with unlimited params that flexable enought for all people.

I'm wondering if we could have some design like OpenTelemetry Collector Contrib, which holds a lot of different receiver / processor / exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter

Further more, new exporter could be added via `git submodule`. I think the following step is preferred, in order to apply specific exporter to the latest DeepFlow release, without putting codes to DeepFlow repo:
1. `git clone` DeepFlow repo.
2. `git submodule pull` a external / internal / private exporter to the exporter folder, e.g. `./server/ingester/flow_log/exporter/my_wonderful_exporter/`
3. make build
4. ... (push image / restart deepflow server with new image tag)

So, exporters that not universal enough could be hold in private repo / company gitlab, and easily integrate with the latest version of DeepFlow. Those exporter could be managed more easier, and donate to the community at the right time, rather than **designing a universal exporter at the very begining but failed**.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/deepflowio/deepflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by lzf575** [comment]
目前可以参照 otlp exporter 来实现增加新的exporter， 需要增加修改的代码都在 [server/ingester/flow_log/exporters/](https://github.com/deepflowio/deepflow/tree/main/server/ingester/flow_log/exporters) 下
1. /exporters.go  增加对新增exporter的启动
2. /config/config.go  增加新exporter 的配置项
3. 新建目录 xxx_exporter 并实现 exporter.go 中的Exporter  interface
